jupyter.Rmd

```diff
@@ -104,8 +104,7 @@ To run a code cell independently, the cell needs to first be activated. This
 is done by clicking on it with the cursor. Jupyter will indicate a cell has been
 activated by highlighting it with a blue rectangle to its left. After the cell
 has been activated (Figure \@ref(fig:activate-and-run-button)), the cell can be run by either pressing the **Run** (`r fa("play", height = "11px")`) 
-button in the toolbar, or by using a keyboard shortcut of 
-`Shift + Enter`.
+button in the toolbar, or by using the keyboard shortcut `Shift + Enter`.
```

Comment:
the list on pg 350-351 has one bullet spilling onto the next pg

Reply:
done